chef/automate
# inspec-agent node add cleanup

### :nut_and_bolt: Description: What code changed, and why?
1) do not add node from the resolver. it will get added to the db when
it is ingested
2) make sure the manager (type and id) is included in the process node
func to ensure we correctly display data for the nodes

I was seeing some weird messages in the logs, so I decided to try to track them down and read through some of the code. I discovered we were doing an `add node` function from the inspec-agent resolver that was calling ProcessNode. But that call was getting rejected b/c ProcessNode is supposed to be called with some run or scan data. 
The node will eventually get added to the db upon ingestion, so it is unnecessary to do this call.
Also making sure the manager id/type get included in the process node function so the nodes are correctly associated with a manager. 

### :chains: Related Resources
N/A

### :+1: Definition of Done
no more of these msgs in the logs: 
<img width="1440" alt="Screen Shot 2019-11-25 at 15 36 29" src="https://user-images.githubusercontent.com/10341541/69988194-6495a980-14fe-11ea-8896-b5c259964a7f.png">

### :athletic_shoe: How to Build and Test the Change
- add a node in ec2. make sure you can ssh into it.
- create a node integration (aws-ec2), create a node credential
- ensure you do the whole "tags - node credentials" thing 
- delete all the nodes assoc with the integration
- run a scan job on the integration
- expect the node you can ssh into to show up in the scan job nodes with the correct information

### :white_check_mark: Checklist

- [n/a] Docs added/updated?

### :camera: Screenshots, if applicable
